pallets/inflation/Cargo.toml

```
@@ -0,0 +1,50 @@
+[package]
+name = "pallet-inflation"
+version = "4.0.0-dev"
+description = "t3rn pallet for handling token inflation and staking rewards"
+authors = ["t3rn devs <team@t3rn.io"]
+homepage = "https://t3rn.io/"
+license = "Apache 2.0"
+edition = "2018"
+publish = false
+repository = "https://github.com/t3rn/t3rn"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = [
+    "derive",
+] }
+serde = { version = "1.0.136", default-features = false, features = ["derive"] }
+scale-info = { version = "1.0", default-features = false, features = ["derive"] }
+frame-support = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", default-features = false }
+frame-system = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", default-features = false }
+frame-benchmarking = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", default-features = false, optional = true }
+sp-runtime = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13" }
+
+## substrate deps
+sp-std = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13" }
+pallet-balances = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", default-features = false }
+pallet-staking = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", default-features = false }
+
+fixed = { git = "https://github.com/encointer/substrate-fixed", package = "substrate-fixed", default-features = false }
+
+[dev-dependencies]
+sp-core = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13" }
+sp-io = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13" }
+
+[features]
+default = ["std"]
+std = [
+    "codec/std",
```

We should add this here
```suggestion
    "codec/std",
    "serde/std",
```

---

Fixing, thanks